python/src/trezorlib/cli/__init__.py

```diff
@@ -17,34 +17,37 @@
 import functools
 import sys
 from contextlib import contextmanager
+from typing import Callable, Optional
 
 import click
 
 from .. import exceptions
 from ..client import TrezorClient
-from ..transport import get_transport
+from ..transport import Transport, get_transport
 from ..ui import ClickUI
 
 
 class ChoiceType(click.Choice):
-    def __init__(self, typemap):
+    def __init__(self, typemap: dict) -> None:
```

Comment:
Yes, it is more appropriate, as it will check if the keys are not some non-string type.

If we declare `Any` as a value, we could also give `Any` returning value to the `convert()` function just below

Reply:
Done in 3c5b9b6